scripts/portfolioView.js

```diff
@@ -0,0 +1,91 @@
+(function(module){
+  var portfolioView = {};
+  var $headerPosition = $('header').offset().top;
+
+  // helper functions
+  // toggle sticky header class based on scroll position
+  function stickyHeader(){
+    if($(this).scrollTop() > $headerPosition){
+      $('header').addClass('sticky');
+    } else {
+      $('header').removeClass('sticky');
+    }
+  };
+
+  // on click build detailed view of project
+  portfolioView.detailHandler = function (){
+    $('#portfolio').on('click','.project-preview',function(){
+      $('.project-details').insertAfter($(this));
+      var $clickedItem = $(this).data('project');
+      PortfolioItem.all.forEach(function(a){
+        if(a.project === $clickedItem){
+          $('.project-details').html(a.updateDetailModal());
+          setTimeout(function(){
```

Comment:
My immediate thought is to put insertAfter into a function that takes a callback, so you can call that function and pass in a callback as you do so. 

Also, you might want to look into the JS array method .some(), which could help you with that PortfolioItem.all.forEach loop. (I did something similar, and some() was useful there)